### DISCUSSION TITLE
Ability to set restrictions on outgoing payments

### DISCUSSION THREAD

**Discussion by statusquont** [ACINQ/phoenixd#101] [Ideas]
It might be nice if there were a way that you could set restrictions on outgoing payments somehow. Maybe even lock in a way that funds could only ever move to a hardcoded Bolt12 offer. Not sure how this would be done technically, just tossing out an idea here.

I'm thinking about situations where if you're running phoenixd in a server environment, and/or if other people (like devs) have access to your server. I realize that you shouldn't be working with people you don't trust, but hey, things happen. 

I guess those same people would still potentially have access to the seed phrase on the server anyways since it's in the config file they can probably get to. On that note, are there best practices documented anywhere, on how to work with 3rd parties while still not giving them keys to the kingdom, if even possible?

**Comment by pm47** [comment]
We have recently introduced a limited access api user: https://github.com/ACINQ/phoenixd/pull/84. But indeed it's moot if people have access to the server and the seed. There is nothing we can do in that case.
